cgmanifest.json

```diff
@@ -1580,6 +1580,15 @@
         }
       }
     },
+    {"component": {
```

Comment:
Nit: Put "component" on a new line to match the format of others listed in this file

Reply:
Done